gamemode/modules/language/sh_language.lua

```diff
@@ -1,10 +1,18 @@
 local rp_languages = {}
-local selectedLanguage = GetConVar("gmod_language"):GetString() -- Switch language by setting gmod_language to another language
+local selectedLanguage = "en" -- Switch language by setting gmod_language to another language
```

Comment:
I think even if the convar is empty, we should still try to use it. That way it will still work if some other server managed to configure it differently.